### PR TITLE
Add Error Messages To Reverse Trace

### PR DESCRIPTION
**Pull request by johnsonw**
srcmap-reverse does not currently include the error message in the
  reverse trace. Make sure to include error messages, even with multiple
  lines, in the reversed trace so that the error message can be seen in
  the logs.

Signed-off-by: Will Johnson <william.c.johnson@intel.com>

### CODE REVIEW

File: source/index.js [hunk 1]
```diff
@@ -42,8 +42,16 @@ if (cluster.isMaster) {
...
           if (process.send) process.send({ error: e });
         })
-        .each(line => {
-          if (process.send) process.send({ line });
+        .each((newLine: string[]) => {
+          if (!process.send) return;
+
+          let lineToSend;
+          if (newLine.length === 0) lineToSend = [];
+          else if (newLine[0].length === 0 && line && line.length > 0)
```
Comment (jgrund) [inline source/index.js]: Is line here the error message?

Comment (johnsonw) [reply source/index.js]: line could be:
1. ['Error: Oh no source maps!']
2. [at ... line ...]
3. null

If we try to reverse "Error: Oh no source maps!" it will be returned as an empty string. In this case, we want to push the original value of line (Error: Oh no source maps!).

Comment (johnsonw) [reply source/index.js]: I added a few more test cases to show the null case, the error message case, and the line to be traced case.
